src/domain/services/notice_service.py

```diff
@@ -17,27 +18,27 @@ async def service_read_notices(page: int, limit: int, db: Session):
 
     try:
         notices = db.execute(stmt).scalars().all()
+
         if not notices:
             raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Notices not found")
-    except Exception as e:
-        raise HTTPException(
```

Comment:
elif not total:
   raise HTTPException(status_code = staus.HTTP_500_SERVER_ERROR, detail="Fetch incorrect total value"